packages/addon/lua/html/markdown_popup.html.lua

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
Same thing for the rest of these files, should they really be named `markdown_popup`?